scaler/handlers.go

```diff
@@ -151,15 +160,16 @@ func (e *impl) GetMetricSpec(
 			)
 			return nil, err
 		}
+		host = "http-requests"
```

Comment:
```suggestion
```

Reply:
Same as above.

Reply:
let's use the constant
```suggestion
		host = httpRequests
```